libs/vertexai/langchain_google_vertexai/vectorstores/_searcher.py

```diff
@@ -30,17 +31,33 @@ class Searcher(ABC):
     def find_neighbors(
         self,
         embeddings: List[List[float]],
+        sparse_embeddings: Optional[List[Dict[str, List[int] | List[float]]]] = None,
```

Comment:
can we make new arguments be explicitly called by name to ensure backwards compatibility, please?
so smth like `..., *, sparse_embeddings: ...`?

Reply:
done